tests/core/tests/admin_integration/test_export.py

```diff
@@ -28,16 +28,23 @@
 
 
 class ExportAdminIntegrationTest(AdminTestMixin, TestCase):
+    def setUp(self) -> None:
```

Comment:
I think it would be a better idea to revert the changes in this Test add a new Test class specifically for this change.  It wouldn't need to duplicate all the tests in `ExportAdminIntegrationTest` but just tests the integration of the changes you have added?

I would like there to be some tests retained for the vanilla `ExportForm`.

Reply:
When using `SelectableFieldsExportForm`, every export operation requires at least one field to be present in form.data. 

So I could revert this change, set `ExportViewMixin.form_class` and `ExportMixin.export_form_class` as `ExportForm` (like before) for the test case then tests would pass.

We ideally still want to run same tests for `SelectableFieldsExportForm`. Creating another test case by inheriting `ExportAdminIntegrationTest`, override the failing tests (which are present in the commit). But I'm not sure if that would be the correct approach

Reply:
`SelectableFieldsExportForm` derives from `ExportForm`, `ExportAdminIntegrationTest` runs tests using `SelectableFieldsExportForm`, so I would say we don't have admin integration test specifically using `ExportForm`.

I would be happy to have export integration test for both `ExportForm` and `SelectableFieldsExportForm` but I'm not sure how I would do it without a lot of duplicate code.

Reply:
OK - happy to leave it as is, thanks.